packages/components/src/navigator/index.tsx

```diff
@@ -0,0 +1,134 @@
+/**
+ * Internal dependencies
+ */
+import { Navigator as TopLevelNavigator } from './navigator';
+import { NavigatorScreen } from './navigator-screen';
+import { NavigatorButton } from './navigator-button';
+import { NavigatorBackButton } from './navigator-back-button';
+export { useNavigator } from './use-navigator';
+
+/**
+ * The `Navigator` component allows rendering nested views/panels/menus
+ * (via the `Navigator.Screen` component) and navigate between these different
+ * view (via the `Navigator.Button` and `Navigator.BackButton` components or the
+ * `useNavigator` hook).
+ *
+ * @example
+ * ```jsx
+ * import { Navigator } from '@wordpress/components';
+ *
+ * const MyNavigation = () => (
+ *   <Navigator initialPath="/">
+ *     <Navigator.Screen path="/">
+ *       <p>This is the home screen.</p>
+ *        <Navigator.Button path="/child">
+ *          Navigate to child screen.
+ *       </Navigator.Button>
+ *     </Navigator.Screen>
+ *
+ *     <Navigator.Screen path="/child">
+ *       <p>This is the child screen.</p>
+ *       <Navigator.BackButton>
+ *         Go back
+ *       </Navigator.BackButton>
+ *     </Navigator.Screen>
+ *   </Navigator>
+ * );
+ * ```
+ */
+export const Navigator = Object.assign( TopLevelNavigator, {
+	dislayName: 'Navigator',
```

Comment:
```suggestion
```

I can confirm this is unnecessary because it's misspelled 😄

Reply:
Lol. That's one way to test components out

Reply:
Done https://github.com/WordPress/gutenberg/pull/64613/commits/81931212710d2112376156173907acaa10d7158a